src/components/TooltipText/TooltipText.tsx

```diff
@@ -0,0 +1,17 @@
+import { Tooltip } from '@patternfly/react-core';
+import { InfoCircleIcon } from '@patternfly/react-icons';
+
+interface ITooltipText {
+  tooltip: string;
+}
+
+export const TooltipText = ({ tooltip, children }: React.PropsWithChildren<ITooltipText>) => {
```

Comment:
1) children could be optional, so something like `children && ... `could be used

2) icon could be passed as component property and `<InfoCircleIcon />` would be the default value, so your component can be reused also on the other places (see for example `BuildStatusIcon.tsx`)